app/controllers/errors_controller.rb

```diff
@@ -50,41 +48,31 @@ def notify_subscribers
   end
 
   def resolve
-    #has values when we resolve issues through the error sidebar
-    errors_per_page = params[:per_page] || 5
-    page = params[:page]
-    if params[:error_ids]
-      ids = params[:error_ids]
-      resolved = to_boolean(params[:resolved]) if params[:resolved]
-      resolve_issues(ids, resolved, errors_per_page, page, @error)
-    elsif params[:individual_resolve]
-      ids = [@error.id]
-      resolved = !@error.resolved_at.nil?
-      resolve_issues(ids, resolved, errors_per_page, page, @error)
-    else
-      raise 'Could not find error!'
-    end
+    resolve_issues(params[:error_ids], true, params[:per_page], params[:page], @error.id)
+    render 'errors/resolve'
+  end
+
+  def unresolve
+    resolve_issues(params[:error_ids], false, params[:per_page], params[:page], @error.id)
+    render 'errors/resolve'
```

Comment:
I can remove it from the 'resolve' method since its done by default but I need the 'unresolve' method to render the same file (resolve.js.haml) not unresolve.js.haml. Is there a better way to do this?